### PR TITLE
Added Mummy

### PR DESCRIPTION
**Pull request by starryburst**
Tested:

Visitors die when visiting a dead mummy (Amnesiac)
Visitors don't die when visiting an alive mummy
Roleblock while dead works

### CODE REVIEW

**Comment by pikulet** [comment]
Can KillVisitorsWhileDead make use of the babushka card with whileDead true?

File: Games/types/Mafia/roles/cards/KillVisitorsWhileDead.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                        this.actor.role.data.visitors = [];
+                    }
+
+                }
+            }
+        ];
+
+        this.listeners = {
+            "death": function (player, killer, deathType) {
+                if (player == this.player && this.data.playerToReveal)
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/KillVisitorsWhileDead.js:52]: This is from oracle code, don't think it should be here

File: Games/types/Mafia/roles/cards/KillVisitorsWhileDead.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+module.exports = class KillVisitorsWhileDead extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.actions = [
+            {
+                priority: PRIORITY_KILL_VISITORS_ENQUEUE,
+                run: function () {
+                    if (!this.actor.role.data.dead)
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/KillVisitorsWhileDead.js:13]: I believe you can check this.actor.alive or some variant of it

**Comment by starryburst** [comment]
Changes are buggy, will fix then reopen

**Pull request closed without merging**